apps/admin/backend/src/cvr_files.ts

```diff
@@ -436,13 +438,14 @@ export function getAddCastVoteRecordReportErrorMessage(
       return 'Unable to parse cast vote record report, it may be malformed.';
     case 'invalid-report-file-mode':
       if (error.currentFileMode === Admin.CvrFileMode.Official) {
-        return `You are currently tabulating official results but the selected cast vote record report report contains test results.`;
+        return `You are currently tabulating official results but the selected cast vote record report contains test results.`;
       }
       return `You are currently tabulating test results but the selected cast vote record report contains official results.`;
     case 'invalid-cast-vote-record': {
       const messageBase = `Found an invalid cast vote record at index ${error.index} in the current report.`;
       const messageDetail = (() => {
         const subErrorType = error.error;
+        /* istanbul ignore next  - write testing when error handling requirements and implementation harden */
```

Comment:
Happy to go back and write testing for this but it seemed like a lot of branches that would require a bunch of copy-pasted tests that don't actually verify much. I wasn't sure about this error handling approach in general so punted.